.circleci/config.yml

```diff
@@ -57,9 +37,28 @@ jobs:
       - run: cd /go/src/istio.io/istio; dep ensure
       - run: cd /go/src/istio.io/istio; go build -i ./pilot/... ./mixer/... ./broker/...
```

Comment:
take a look at setup remote docker. it allows you to build docker images
you can build and then push docker images and then you can start up the e2e tests, while you can run unit tests simultaneously